internal/util/util.go

```diff
@@ -80,3 +80,15 @@ func ResolveConfigFilePaths(config *config.MountConfig) (err error) {
 	}
 	return
 }
+
+// MibToBytes returns the bytes equivalent
+// of given no.s of MiBs (Mibi Bytes).
+// For reference, each MiB = 2^20 bytes.
+// It supports only upto 2^44-1 MiBs (~4 Tebi MiBs, or ~4 Ebi bytes)
+// as inputs, and panics for higher inputs.
+func MibToBytes(mib uint64) uint64 {
+	if mib > 0xFFFFFFFFFFF {
```

Comment:
Use some pre-defined constant instead of a weird hex constant. MAX_UINT64 or something.

Reply:
There is no standard golng constant for this. I've defined constant MaxMiBsInMaxUint64 above for it now.